apis/cassandra/v1beta1/cassandradatacenter_webhook.go

```diff
@@ -114,6 +119,11 @@ func ValidateSingleDatacenter(dc CassandraDatacenter) error {
 		}
 	}
 
+	err := ValidateServiceLabelsAndAnnotations(dc)
```

Comment:
Nit, usually this is just written with `if err := ..; err != nil {` instead of two pieces.